### PR TITLE
Paths update script

### PR DESCRIPTION
**Pull request by am-e**
Perl script to update Javascript and CSS paths in HTML.

### CODE REVIEW

**Comment by mrpatrickwest** [comment]
If I had to guess I'd say that you created this branch from your project browser branch? I would expect to see the script in here and not your project browser files.

File: scripts/switch_paths.pl [hunk 1]
```diff
@@ -0,0 +1,22 @@
+$num_args = $#ARGV + 1;
+if ($num_args != 2) {
+    print "\nUsage: switch_dev_prod.pl dev/prod filename\n";
+    exit;
```
Comment (mrpatrickwest) [inline scripts/switch_paths.pl:4]: Add a print here to describe what the script does. For completeness.

File: scripts/switch_paths.pl [hunk 2]
```diff
@@ -0,0 +1,22 @@
+$num_args = $#ARGV + 1;
+if ($num_args != 2) {
+    print "\nUsage: switch_dev_prod.pl dev/prod filename\n";
```
Comment (mrpatrickwest) [inline scripts/switch_paths.pl]: Thinking you mean dev|prod and not dev/prod

**Comment by am-e** [comment]
I had initially added the script under the project-browser, then realized the error of my ways and removed the script. I then created a new branch online, checked it out locally, added the script and pushed the changes.

**Comment by zednis** [comment]
Hi Ahmed,  based on the branch network diagram (see Graphs on the right-hand side in github web UI) the Paths-update-script branch was branched off the project-browser branch and as such includes in it's history all the previous commits to the project-browser branch.

merging this branch to Master will also merge all the previous commits made to project-browser.

**Comment by zednis** [comment]
My recommendation is we close this pull request without merging, delete the Paths-update-script branch, and create a new path-script branch based on the current master branch.  We will then be able to merge the path script into master fully independent of the project-browser updates.

**Comment by am-e** [comment]
Yes, I agree. Sorry about that, I'll be more careful in the future. If we are all in agreement, I'll go ahead and implement what you suggested.

**Comment by zednis** [comment]
:+1:

**Comment by mrpatrickwest** [comment]
:+1:

**Pull request closed without merging**